WooCommerce/Classes/POS/Utils/POSFontStyle.swift

```diff
@@ -5,35 +5,32 @@ import SwiftUI
 enum POSFontStyle {
     case posTitleRegular
     case posTitleEmphasized
-    case posBodyRegular
-    case posBodyEmphasized
-    case posLargeDetailEmphasized
-    case posLargeDetailRegular
-    case posDetailLight
-    case posDetailRegular
-    case posDetailEmphasized
+    case posBodyLargeRegular
+    case posBodyLargeEmphasized
```

Comment:
Since the new system uses `bold` instead of `emphasized` we could align on the naming as well.

```suggestion
    case posBodyLargeBold
```